msgs_stack/kraken_msgs/msg/imuData_new.msg

```diff
@@ -0,0 +1,7 @@
+Header header
```

Comment:
@athityakumar This file is not required. You can directly use the sensor messages from ROS standard library by doing `import ros_msg from rospy` or something of that sort. I don't remember the exact line, but search for it.